daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -71,7 +71,7 @@ type logStream struct {
 	logStreamName      string
 	logGroupName       string
 	logCreateGroup     bool
-	logNonBlocking     bool
+	logCreateStream    bool
```

Comment:
somehow i messed up the rebase and added this... not sure how because the base 20.10 branch doesn't have it

Reply:
fixed